src/ring_theory/algebra.lean

```diff
@@ -205,12 +205,18 @@ protected def id : A →ₐ[R] A :=
 { to_fun := id, commutes' := λ _, rfl }
 variables {R A}
 
+@[simp] lemma id_to_linear_map :
+  (alg_hom.id R A).to_linear_map = @linear_map.id α β _ _ _
```

Comment:
```suggestion
  (alg_hom.id R A).to_linear_map = @linear_map.id R A _ _ _
```
Just guessing that it should look like this.

Reply:
Thanks Chris, it's hard to write code without a local compiler helping me.